From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: hayaksi1 <193020925+hayaksi1@users.noreply.github.com>
Date: Thu, 27 Aug 2026 20:25:46 +0300
Subject: [PATCH 08/10] Collapse the read-receipt import oxfmt wanted on one
 line

Reflowing this by hand after the vitest hook import was added left the
specifier list wrapped when it now fits the line budget, which is the
sole reason the formatter gate was red.
---
 .../src/components/views/rooms/ReadReceiptGroup.test.tsx   | 7 +------
 1 file changed, 1 insertion(+), 6 deletions(-)

diff --git a/apps/web/src/components/views/rooms/ReadReceiptGroup.test.tsx b/apps/web/src/components/views/rooms/ReadReceiptGroup.test.tsx
index ada64802c7..638a8732f7 100644
--- a/apps/web/src/components/views/rooms/ReadReceiptGroup.test.tsx
+++ b/apps/web/src/components/views/rooms/ReadReceiptGroup.test.tsx
@@ -15,12 +15,7 @@ import { RoomMember } from "matrix-js-sdk/src/matrix";
 import userEvent from "@testing-library/user-event";
 import { stubClient } from "test-utils";
 
-import {
-    determineAvatarPosition,
-    ReadReceiptGroup,
-    ReadReceiptPerson,
-    readReceiptTooltip,
-} from "./ReadReceiptGroup";
+import { determineAvatarPosition, ReadReceiptGroup, ReadReceiptPerson, readReceiptTooltip } from "./ReadReceiptGroup";
 import * as languageSettings from "../../../i18n/settings";
 import dispatcher from "../../../dispatcher/dispatcher";
 import { Action } from "../../../dispatcher/actions";
